pkg/sip/room.go

```diff
@@ -96,7 +96,7 @@ func NewRoom(log logger.Logger, st *RoomStats) *Room {
 	}
 	r := &Room{log: log, stats: st, out: msdk.NewSwitchWriter(RoomSampleRate)}
 	out := newMediaWriterCount(r.out, &st.OutputFrames, &st.OutputSamples)
-	r.mix = mixer.NewMixer(out, rtp.DefFrameDur, &st.Mixer)
+	r.mix, _ = mixer.NewMixer(out, rtp.DefFrameDur, &st.Mixer, 1)
```

Comment:
We should at least panic here in case of an error. Debugging `r.mix == nil` deeper in the code is never fun 😅

Reply:
yep, my bad.